### DISCUSSION TITLE
Prevent vendor.css from being included?

### DISCUSSION THREAD

**Discussion by digitalmagnets** [quasarframework/quasar#14943] [Ideas / Proposals]
Is there a way to prevent vendor.css from being included?

I would like to include a manually edited version (removing bg colours etc. I'm not going to use). 

I'd also like to experiment with manually including required styles with relevant components, to get the first load size down further.

**Comment by suleimanas** [comment]
try PurgeCSS you can safelist or blocklist anything / I use it to remove some classes if I'm sure that i'll never use them
